.npmignore

```diff
@@ -1,3 +1,2 @@
 test
-.editorconfig
```

Comment:
Why would you delete the editorconfig file?

Reply:
I can leave it if you prefer

Reply:
I do - editorconfigs are a good way to stay in sync with editor-level settings like indentation etc no matter which text editor you are using. There is an argument to be made for prettier superseding it in this case though, as long as prettier running per-commit is made mandatory.